src/mlagility/helpers/labels.py

```diff
@@ -0,0 +1,80 @@
+import os
+from typing import Dict, List
+
+
+def load_from_file(file_path: str) -> Dict[str, List[str]]:
+    """
+    This function extracts labels from a Python file.
+    Labels must be in the first line of a Python file and start with "# labels: "
+    Each label must have the format "key::value1,value2,..."
+
+    Example:
+        "# labels: author:google test_group:daily,monthly"
+    """
+    # Labels must be in the first line of the python file
+    with open(file_path, encoding="utf-8") as f:
+        first_line = f.readline()
+
+    if "# labels:" in first_line:
+
```

Comment:
style: remove empty line

Reply:
removed! We should probably modify lint to also catch that.